src/model/ProductCategoryModel.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
+ */
+package model;
+
+import java.util.Date;
+
+/**
+ *
+ * @author Govind Kumar
+ */ 
+public class ProductCategoryModel {
+    private String category_name;
+    private String category_code;
+    private String created_date;
+    private Integer id;
+    private boolean active;
+    private String created_by;
+    private String updated_by;
+
```

Comment:
do not write Model word in the class name and please follow camelCase notation for variable names and also 
connect with @TouqeerAli for setting these fields
  private String created_date;
    private Integer id;
    private boolean active;
    private String created_by;
    private String updated_by;